{
  "abstract": [
    {
      "text": "An unknown error occurred.",
      "type": "text"
    }
  ],
  "hierarchy": {
    "paths": [
      [
        "doc://com.apple.documentation/documentation/technologies",
        "doc://com.apple.callkit/documentation/CallKit",
        "doc://com.apple.callkit/documentation/CallKit/CXErrorCodeIncomingCallError-swift.struct"
      ]
    ]
  },
  "identifier": {
    "interfaceLanguage": "swift",
    "url": "doc://com.apple.callkit/documentation/CallKit/CXErrorCodeIncomingCallError-swift.struct/unknown"
  },
  "kind": "symbol",
  "legalNotices": {
    "copyright": "Copyright &copy; 2025 Apple Inc. All rights reserved.",
    "privacyPolicy": "https://www.apple.com/privacy/privacy-policy",
    "termsOfUse": "https://www.apple.com/legal/internet-services/terms/site.html"
  },
  "metadata": {
    "externalID": "s:SC28CXErrorCodeIncomingCallErrorLeV7unknownSoAAVvpZ",
    "fragments": [
      {
        "kind": "keyword",
        "text": "static"
      },
      {
        "kind": "text",
        "text": " "
      },
      {
        "kind": "keyword",
        "text": "var"
      },
      {
        "kind": "text",
        "text": " "
      },
      {
        "kind": "identifier",
        "text": "unknown"
      },
      {
        "kind": "text",
        "text": ": "
      },
      {
        "kind": "typeIdentifier",
        "preciseIdentifier": "s:SC28CXErrorCodeIncomingCallErrorLeV",
        "text": "CXErrorCodeIncomingCallError"
      },
      {
        "kind": "text",
        "text": "."
      },
      {
        "kind": "typeIdentifier",
        "preciseIdentifier": "c:@E@CXErrorCodeIncomingCallError",
        "text": "Code"
      }
    ],
    "modules": [
      {
        "name": "CallKit"
      }
    ],
    "platforms": [
      {
        "beta": false,
        "deprecated": false,
        "introducedAt": "10.0",
        "name": "iOS",
        "unavailable": false
      },
      {
        "beta": false,
        "deprecated": false,
        "introducedAt": "10.0",
        "name": "iPadOS",
        "unavailable": false
      },
      {
        "beta": false,
        "deprecated": false,
        "introducedAt": "10.0",
        "name": "Mac Catalyst",
        "unavailable": false
      },
      {
        "beta": false,
        "deprecated": false,
        "introducedAt": "1.0",
        "name": "visionOS",
        "unavailable": false
      },
      {
        "beta": false,
        "deprecated": false,
        "introducedAt": "9.0",
        "name": "watchOS",
        "unavailable": false
      }
    ],
    "role": "symbol",
    "roleHeading": "Type Property",
    "symbolKind": "property",
    "title": "unknown"
  },
  "primaryContentSections": [
    {
      "declarations": [
        {
          "languages": [
            "swift"
          ],
          "platforms": [
            "iOS",
            "iPadOS",
            "Mac Catalyst",
            "visionOS",
            "watchOS"
          ],
          "tokens": [
            {
              "kind": "keyword",
              "text": "static"
            },
            {
              "kind": "text",
              "text": " "
            },
            {
              "kind": "keyword",
              "text": "var"
            },
            {
              "kind": "text",
              "text": " "
            },
            {
              "kind": "identifier",
              "text": "unknown"
            },
            {
              "kind": "text",
              "text": ": "
            },
            {
              "identifier": "doc://com.apple.callkit/documentation/CallKit/CXErrorCodeIncomingCallError-swift.struct",
              "kind": "typeIdentifier",
              "preciseIdentifier": "s:SC28CXErrorCodeIncomingCallErrorLeV",
              "text": "CXErrorCodeIncomingCallError"
            },
            {
              "kind": "text",
              "text": "."
            },
            {
              "identifier": "doc://com.apple.callkit/documentation/CallKit/CXErrorCodeIncomingCallError-swift.struct/Code",
              "kind": "typeIdentifier",
              "preciseIdentifier": "c:@E@CXErrorCodeIncomingCallError",
              "text": "Code"
            },
            {
              "kind": "text",
              "text": " { "
            },
            {
              "kind": "keyword",
              "text": "get"
            },
            {
              "kind": "text",
              "text": " }"
            }
          ]
        }
      ],
      "kind": "declarations"
    }
  ],
  "references": {
    "doc://com.apple.callkit/documentation/CallKit": {
      "abstract": [
        {
          "text": "Display the system-calling UI for your appâ€™s VoIP services, and coordinate your calling services with other apps and the system.",
          "type": "text"
        }
      ],
      "identifier": "doc://com.apple.callkit/documentation/CallKit",
      "kind": "symbol",
      "role": "collection",
      "title": "CallKit",
      "type": "topic",
      "url": "/documentation/callkit"
    },
    "doc://com.apple.callkit/documentation/CallKit/CXErrorCodeIncomingCallError-swift.struct": {
      "abstract": [
        {
          "text": "Codes for errors that occur during incoming calls.",
          "type": "text"
        }
      ],
      "fragments": [
        {
          "kind": "keyword",
          "text": "struct"
        },
        {
          "kind": "text",
          "text": " "
        },
        {
          "kind": "identifier",
          "text": "CXErrorCodeIncomingCallError"
        }
      ],
      "identifier": "doc://com.apple.callkit/documentation/CallKit/CXErrorCodeIncomingCallError-swift.struct",
      "kind": "symbol",
      "navigatorTitle": [
        {
          "kind": "identifier",
          "text": "CXErrorCodeIncomingCallError"
        }
      ],
      "role": "symbol",
      "title": "CXErrorCodeIncomingCallError",
      "type": "topic",
      "url": "/documentation/callkit/cxerrorcodeincomingcallerror-swift.struct"
    },
    "doc://com.apple.callkit/documentation/CallKit/CXErrorCodeIncomingCallError-swift.struct/Code": {
      "abstract": [
        {
          "text": "Codes for errors that occur during incoming calls.",
          "type": "text"
        }
      ],
      "fragments": [
        {
          "kind": "keyword",
          "text": "enum"
        },
        {
          "kind": "text",
          "text": " "
        },
        {
          "kind": "identifier",
          "text": "Code"
        }
      ],
      "identifier": "doc://com.apple.callkit/documentation/CallKit/CXErrorCodeIncomingCallError-swift.struct/Code",
      "kind": "symbol",
      "navigatorTitle": [
        {
          "kind": "identifier",
          "text": "Code"
        }
      ],
      "role": "symbol",
      "title": "CXErrorCodeIncomingCallError.Code",
      "type": "topic",
      "url": "/documentation/callkit/cxerrorcodeincomingcallerror-swift.struct/code"
    },
    "doc://com.apple.callkit/documentation/CallKit/CXErrorCodeIncomingCallError-swift.struct/callUUIDAlreadyExists": {
      "abstract": [
        {
          "text": "The incoming call UUID already exists.",
          "type": "text"
        }
      ],
      "fragments": [
        {
          "kind": "keyword",
          "text": "static"
        },
        {
          "kind": "text",
          "text": " "
        },
        {
          "kind": "keyword",
          "text": "var"
        },
        {
          "kind": "text",
          "text": " "
        },
        {
          "kind": "identifier",
          "text": "callUUIDAlreadyExists"
        },
        {
          "kind": "text",
          "text": ": "
        },
        {
          "kind": "typeIdentifier",
          "preciseIdentifier": "s:SC28CXErrorCodeIncomingCallErrorLeV",
          "text": "CXErrorCodeIncomingCallError"
        },
        {
          "kind": "text",
          "text": "."
        },
        {
          "kind": "typeIdentifier",
          "preciseIdentifier": "c:@E@CXErrorCodeIncomingCallError",
          "text": "Code"
        }
      ],
      "identifier": "doc://com.apple.callkit/documentation/CallKit/CXErrorCodeIncomingCallError-swift.struct/callUUIDAlreadyExists",
      "kind": "symbol",
      "role": "symbol",
      "title": "callUUIDAlreadyExists",
      "type": "topic",
      "url": "/documentation/callkit/cxerrorcodeincomingcallerror-swift.struct/calluuidalreadyexists"
    },
    "doc://com.apple.callkit/documentation/CallKit/CXErrorCodeIncomingCallError-swift.struct/filteredByBlockList": {
      "abstract": [
        {
          "text": "The system is filtering the incoming call because the user is blocking it.",
          "type": "text"
        }
      ],
      "fragments": [
        {
          "kind": "keyword",
          "text": "static"
        },
        {
          "kind": "text",
          "text": " "
        },
        {
          "kind": "keyword",
          "text": "var"
        },
        {
          "kind": "text",
          "text": " "
        },
        {
          "kind": "identifier",
          "text": "filteredByBlockList"
        },
        {
          "kind": "text",
          "text": ": "
        },
        {
          "kind": "typeIdentifier",
          "preciseIdentifier": "s:SC28CXErrorCodeIncomingCallErrorLeV",
          "text": "CXErrorCodeIncomingCallError"
        },
        {
          "kind": "text",
          "text": "."
        },
        {
          "kind": "typeIdentifier",
          "preciseIdentifier": "c:@E@CXErrorCodeIncomingCallError",
          "text": "Code"
        }
      ],
      "identifier": "doc://com.apple.callkit/documentation/CallKit/CXErrorCodeIncomingCallError-swift.struct/filteredByBlockList",
      "kind": "symbol",
      "role": "symbol",
      "title": "filteredByBlockList",
      "type": "topic",
      "url": "/documentation/callkit/cxerrorcodeincomingcallerror-swift.struct/filteredbyblocklist"
    },
    "doc://com.apple.callkit/documentation/CallKit/CXErrorCodeIncomingCallError-swift.struct/filteredByDoNotDisturb": {
      "abstract": [
        {
          "text": "The system is filtering the incoming call because Do Not Disturb is active and the incoming caller isnâ€™t a VIP.",
          "type": "text"
        }
      ],
      "fragments": [
        {
          "kind": "keyword",
          "text": "static"
        },
        {
          "kind": "text",
          "text": " "
        },
        {
          "kind": "keyword",
          "text": "var"
        },
        {
          "kind": "text",
          "text": " "
        },
        {
          "kind": "identifier",
          "text": "filteredByDoNotDisturb"
        },
        {
          "kind": "text",
          "text": ": "
        },
        {
          "kind": "typeIdentifier",
          "preciseIdentifier": "s:SC28CXErrorCodeIncomingCallErrorLeV",
          "text": "CXErrorCodeIncomingCallError"
        },
        {
          "kind": "text",
          "text": "."
        },
        {
          "kind": "typeIdentifier",
          "preciseIdentifier": "c:@E@CXErrorCodeIncomingCallError",
          "text": "Code"
        }
      ],
      "identifier": "doc://com.apple.callkit/documentation/CallKit/CXErrorCodeIncomingCallError-swift.struct/filteredByDoNotDisturb",
      "kind": "symbol",
      "role": "symbol",
      "title": "filteredByDoNotDisturb",
      "type": "topic",
      "url": "/documentation/callkit/cxerrorcodeincomingcallerror-swift.struct/filteredbydonotdisturb"
    },
    "doc://com.apple.callkit/documentation/CallKit/CXErrorCodeIncomingCallError-swift.struct/unentitled": {
      "abstract": [
        {
          "text": "The app doesnâ€™t have the entitlement to receive incoming calls.",
          "type": "text"
        }
      ],
      "fragments": [
        {
          "kind": "keyword",
          "text": "static"
        },
        {
          "kind": "text",
          "text": " "
        },
        {
          "kind": "keyword",
          "text": "var"
        },
        {
          "kind": "text",
          "text": " "
        },
        {
          "kind": "identifier",
          "text": "unentitled"
        },
        {
          "kind": "text",
          "text": ": "
        },
        {
          "kind": "typeIdentifier",
          "preciseIdentifier": "s:SC28CXErrorCodeIncomingCallErrorLeV",
          "text": "CXErrorCodeIncomingCallError"
        },
        {
          "kind": "text",
          "text": "."
        },
        {
          "kind": "typeIdentifier",
          "preciseIdentifier": "c:@E@CXErrorCodeIncomingCallError",
          "text": "Code"
        }
      ],
      "identifier": "doc://com.apple.callkit/documentation/CallKit/CXErrorCodeIncomingCallError-swift.struct/unentitled",
      "kind": "symbol",
      "role": "symbol",
      "title": "unentitled",
      "type": "topic",
      "url": "/documentation/callkit/cxerrorcodeincomingcallerror-swift.struct/unentitled"
    },
    "doc://com.apple.callkit/documentation/CallKit/CXErrorCodeIncomingCallError-swift.struct/unknown": {
      "abstract": [
        {
          "text": "An unknown error occurred.",
          "type": "text"
        }
      ],
      "fragments": [
        {
          "kind": "keyword",
          "text": "static"
        },
        {
          "kind": "text",
          "text": " "
        },
        {
          "kind": "keyword",
          "text": "var"
        },
        {
          "kind": "text",
          "text": " "
        },
        {
          "kind": "identifier",
          "text": "unknown"
        },
        {
          "kind": "text",
          "text": ": "
        },
        {
          "kind": "typeIdentifier",
          "preciseIdentifier": "s:SC28CXErrorCodeIncomingCallErrorLeV",
          "text": "CXErrorCodeIncomingCallError"
        },
        {
          "kind": "text",
          "text": "."
        },
        {
          "kind": "typeIdentifier",
          "preciseIdentifier": "c:@E@CXErrorCodeIncomingCallError",
          "text": "Code"
        }
      ],
      "identifier": "doc://com.apple.callkit/documentation/CallKit/CXErrorCodeIncomingCallError-swift.struct/unknown",
      "kind": "symbol",
      "role": "symbol",
      "title": "unknown",
      "type": "topic",
      "url": "/documentation/callkit/cxerrorcodeincomingcallerror-swift.struct/unknown"
    },
    "doc://com.apple.documentation/documentation/technologies": {
      "abstract": [
        {
          "text": "",
          "type": "text"
        }
      ],
      "identifier": "doc://com.apple.documentation/documentation/technologies",
      "kind": "technologies",
      "role": "overview",
      "title": "Technologies",
      "type": "topic",
      "url": "/documentation/technologies"
    }
  },
  "schemaVersion": {
    "major": 0,
    "minor": 3,
    "patch": 0
  },
  "sections": [],
  "seeAlsoSections": [
    {
      "anchor": "Errors",
      "generated": true,
      "identifiers": [
        "doc://com.apple.callkit/documentation/CallKit/CXErrorCodeIncomingCallError-swift.struct/callUUIDAlreadyExists",
        "doc://com.apple.callkit/documentation/CallKit/CXErrorCodeIncomingCallError-swift.struct/filteredByBlockList",
        "doc://com.apple.callkit/documentation/CallKit/CXErrorCodeIncomingCallError-swift.struct/filteredByDoNotDisturb",
        "doc://com.apple.callkit/documentation/CallKit/CXErrorCodeIncomingCallError-swift.struct/unentitled"
      ],
      "title": "Errors"
    }
  ],
  "variantOverrides": [
    {
      "patch": [
        {
          "op": "replace",
          "path": "/references/doc:~1~1com.apple.callkit~1documentation~1CallKit~1CXErrorCodeIncomingCallError-swift.struct~1Code/title",
          "value": "CXErrorCodeIncomingCallError"
        },
        {
          "op": "replace",
          "path": "/references/doc:~1~1com.apple.callkit~1documentation~1CallKit~1CXErrorCodeIncomingCallError-swift.struct~1Code/fragments",
          "value": [
            {
              "kind": "identifier",
              "text": "CXErrorCodeIncomingCallError"
            }
          ]
        },
        {
          "op": "replace",
          "path": "/references/doc:~1~1com.apple.callkit~1documentation~1CallKit~1CXErrorCodeIncomingCallError-swift.struct~1Code/navigatorTitle",
          "value": [
            {
              "kind": "identifier",
              "text": "CXErrorCodeIncomingCallError"
            }
          ]
        }
      ],
      "traits": [
        {
          "interfaceLanguage": "occ"
        }
      ]
    }
  ],
  "variants": [
    {
      "paths": [
        "/documentation/callkit/cxerrorcodeincomingcallerror-swift.struct/unknown"
      ],
      "traits": [
        {
          "interfaceLanguage": "swift"
        }
      ]
    }
  ]
}
